src/client/components/Checker.tsx

```diff
@@ -17,12 +17,17 @@ import { CheckboxField, RadioField, NumericField, DateField } from './fields'
 import { LineDisplay } from './displays'
 import * as checker from './../../types/checker'
 import { evaluate } from './../core/evaluator'
+import { unit, Unit } from 'mathjs'
 
 interface CheckerProps {
   config: checker.Checker
 }
 
-type VariableResults = Record<string, string | number>
+type VariableResults = Record<string, string | number | Unit>
+
+function isUnit(output: string | number | Unit): output is Unit {
```

Comment:
might refactor this out?

Reply:
This is refactored out in Preston's latest PR. Can you rebase on `develop` and merge in the changes?